### PR TITLE
fix #15588: eliminate the sprintf error on asserting command output

### LINKED ISSUE #15588: Command test fails with error "sprintf(): Too few arguments" when asserting contents that have a format specifier (e.g. %s)
This is a **bug**.

* CakePHP Version: 4.2.6.
* Platform and Target: PHP 7.4 (`sprintf` still returns `false` when fails)

### What you did
Executed a phpunit test like: `vendor/bin/phpunit tests/TestCase/Command/SomethingCommandTest.php`.

The command outputs json formatted text containing format specifiers such as "%s".

### What happened
The test failed  with error: 
```
1) App\Test\TestCase\Command\SomethingCommandTest::testExecute
TypeError: Return value of Cake\TestSuite\Constraint\Console\ContentsContain::toString() must be of the type string, boolean returned
```

The essential problem is that the test throws `sprintf(): Too few arguments` because my `$this->contents` has format specifiers such as "%s".
```php
// Cake\TestSuite\Constraint\Console\ContentsContain::toString()

    public function toString(): string
    {
        return sprintf('is in %s,' . PHP_EOL . 'actual result:' . PHP_EOL . $this->contents, $this->output);
    }
```

### What you expected to happen

Eliminate the error by moving the sprintf's closing bracket like this:
```php
        return sprintf('is in %s,' . PHP_EOL . 'actual result:' . PHP_EOL, $this->output) . $this->contents;
```

### PR DESCRIPTION
**Pull request by fugaco**

### CODE REVIEW

File: src/TestSuite/Constraint/Console/ContentsContain.php [hunk 1]
```diff
@@ -40,6 +40,6 @@ public function matches($other): bool
      */
     public function toString(): string
     {
-        return sprintf('is in %s,' . PHP_EOL . 'actual result:' . PHP_EOL . $this->contents, $this->output);
+        return sprintf('is in %s,' . PHP_EOL . 'actual result:' . PHP_EOL, $this->output) . $this->contents;
```
Comment (othercorey) [inline src/TestSuite/Constraint/Console/ContentsContain.php:43]: Can we encode the newline with double quotes? `"is in %s,\n..."` ?

Comment (dereuromark) [reply src/TestSuite/Constraint/Console/ContentsContain.php:43]: That aint better, given that the constant is Platform awake imo.

Comment (othercorey) [reply src/TestSuite/Constraint/Console/ContentsContain.php:43]: If it needs the windows-specific end lines, then no worries.

**Comment by markstory** [comment]
Thank you 👍

**Pull request merged**